ansible/roles/debops.postfix/tasks/ldap.yml

```diff
@@ -0,0 +1,59 @@
+---
+
+- name: Make sure 'postfix/ldap' directory exists
+  file:
+    dest: '/etc/postfix/ldap'
+    state: 'directory'
+    owner: 'root'
+    group: 'root'
+    mode: '0755'
+
+- name: Generate LDAP virtual files
+  template:
+    src:   'ldap/postfix__{{ item }}.j2'
+    dest:  '/etc/postfix/ldap/{{ item }}.cf'
+    owner: 'postfix'
+    group: 'postfix'
+    mode:  '0400'
+  loop:
+    - 'virtual_aliases'
+    - 'virtual_recipients'
+    - 'virtual_domains'
+    - 'virtual_senders'
+  no_log: '{{ postfix__no_log|d(True) }}'
+
+- name: "Make sure vmail home directory ({{ postfix__virtual_mailbox_base }}) exists"
+  file:
+    dest: '{{ postfix__virtual_mailbox_base }}'
+    state: 'directory'
+    owner: '{{ postfix__virtual_mail_posix_user }}'
+    group: '{{ postfix__virtual_mail_posix_group }}'
+    mode: '0755'
```

Comment:
The mailbox base is the home directory of the `vmail` UNIX account, right? That one should already exist, you don't need this particular task. Unless you want to make the directory mode more strict, which might be a good idea.